modules/dnn/test/test_layers.cpp

```diff
@@ -1453,4 +1453,86 @@ TEST(Layer_Test_Convolution, relu_fusion)
     normAssert(input, output);
 }
 
+// Check fuse convolution and relu wherein skip eltwise
+TEST(Layers_Test_Subnet, convolutionx2_eltwise_scalex2_relu)
+{
+    Net net;
+    int id_conv[2];
+    int weightsShape[] = {1, 1, 1, 1};
+    Mat weights(4, &weightsShape[0], CV_32F, Scalar(2));
```

Comment:
Inputs are too weak, so testing coverage is very low.

Reply:
@alalek please review test again)